flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -79,9 +90,21 @@ public NetworkEnvironment(NetworkEnvironmentConfiguration config, TaskEventPubli
 
 		this.taskEventPublisher = checkNotNull(taskEventPublisher);
 
+		registerNetworkMetrics(parentMetricGroup);
+
 		isShutdown = false;
 	}
 
+	private void registerNetworkMetrics(ComponentMetricGroup parentMetricGroup) {
```

Comment:
As it is used in constructor, could we convert it into a static method:
`private static void registerNetworkMetrics(ComponentMetricGroup parentMetricGroup, networkBufferPool)`?

Reply:
ok